client/core/core.go

```diff
@@ -2430,9 +2561,23 @@ func (c *Core) authDEX(dc *dexConnection) error {
 	}
 
 	// Set the account as authenticated.
-	log.Debugf("Authenticated connection to %s, %d active matches", dc.acct.host, len(result.Matches))
+	log.Debugf("Authenticated connection to %s, %d active orders, %d active matches",
+		dc.acct.host, len(result.Orders), len(result.Matches))
 	dc.acct.auth()
 
+	// Compare the server-returned active orders with tracked trades, updating
+	// the trade statuses where necessary.
+	unknownOrdersCount, reconciledOrdersCount := dc.reconcileTrades(result.Orders)
+	if unknownOrdersCount > 0 {
+		details := fmt.Sprintf("%d active orders reported by %s were not found.",
+			unknownOrdersCount, dc.acct.host)
+		c.notify(newDEXAuthNote("DEX reported unknown orders", dc.acct.host, false, details, db.Poke))
+	}
+	if reconciledOrdersCount > 0 {
+		details := fmt.Sprintf("Statuses updated for %d orders.", reconciledOrdersCount)
+		c.notify(newDEXAuthNote("Orders reconciled with DEX", dc.acct.host, false, details, db.Poke))
+	}
+
```

Comment:
I'm going to bring it up in Element, and I ran into this problem when implementing `match_status`, but hooking in here to resolve status conflicts is not great, because the server may have cached messages coming that would naturally resolve the conflicts.